src/zap/zaprelocs.cpp

```diff
@@ -107,6 +107,20 @@ void ZapBaseRelocs::WriteReloc(PVOID pSrc, int offset, ZapNode * pTarget, int ta
         // IMAGE_REL_BASED_THUMB_BRANCH24 does not need base reloc entry
         return;
 #endif
+#if defined(_TARGET_ARM64_)
+    case IMAGE_REL_ARM64_BRANCH26:
+        {
+            TADDR pSite = (TADDR)m_pImage->GetBaseAddress() + rva;
+
+            INT64 relOffset = (INT64)(pActualTarget - (pSite + sizeof(INT64)));
+            if (!FitsInRel28(relOffset))
+            {
+                ThrowHR(COR_E_OVERFLOW);
```

Comment:
that is done for arm and only applies for desktop case and not coreclr. 
m_fNGenLastRetry is only valid for desktop and not for coreclr.
So not doing anything here for arm64